lib/loaders/express.loader.ts

```diff
@@ -48,6 +48,14 @@ export class ExpressLoader extends AbstractLoader {
         app.use(express.static(clientPath, options.serveStaticOptions));
         app.get(options.renderPath, renderFn);
       }
+
+      app.use((err: Error, req, res, next: Function) => {
+        if (err?.message?.includes('ENOENT')) {
```

Comment:
Quick question: What happens if the route doesn't exist anywhere in the server? Would an error with `ENOENT` still come back, or does `express` take care of that in some other way?

Reply:
Yes, I will make the change for `fastify.loader.ts`. 
If the route doesn't exist it returns 404

```json
{
  "response":{
    "statusCode":404,
    "message":"ENOENT: no such file or directory, stat '/usr/local/foo/bar/index.html'",
    "error":"Not Found"
  },
  "status":404,
  "message":"ENOENT: no such file or directory, stat '/usr/local/foo/bar/index.html'"
}
```
I am testing my fix with this sample app that has no route but only `ServeStaticModule` pointing to a path that does not exist.

Regarding the release, it's actually a bug fix IMO: it should not return 500 if the files are not found. What do you think?